Doc/library/datetime.rst

```diff
@@ -1485,13 +1485,6 @@ Instance methods:
    instance. The return value is a :class:`float` similar to that
    returned by :func:`time.time`.
 
-   Naive :class:`.datetime` instances are assumed to represent local
```

Comment:
I think we are not strictly using `mktime`, but the general thrust of this is still true, isn't it? We're using the native `localtime_s`, which, IIRC, still has some platform-specific behavior, I think mostly on Windows.

It's also still true that the naïve instances represent local time, so we shouldn't be cutting that part out.